contracts/NodeList.sol

```diff
@@ -0,0 +1,29 @@
+pragma solidity ^0.4.18;
+
+import "@aragon/os/contracts/apps/AragonApp.sol";
+
+contract NodeList is AragonApp {
+
+  event NewMember(address indexed ethNodeAddress, bytes16 ipAddress);
+  event MemberRemoved(address indexed ethNodeAddress, bytes16 ipAddress);
+ 
+  bytes32 constant public ADD_MEMBER = keccak256("ADD_MEMBER");
+  bytes32 constant public DELETE_MEMBER = keccak256("DELETE_MEMBER");
+
+  mapping(bytes16 => address) public nodeList;
+
+  function addMember(address _ethAddr, bytes16 _ip) auth(ADD_MEMBER) external {
```

Comment:
Not a solidity expert just noticed `deleteMember` mentions `ADD_MEMBER`. Shouldn't it do `auth(DELETE_MEMBER)`?

Reply:
Most certainly. good catch!